audit/projects/k8s-staging-cluster-api-nested/description.json

```diff
@@ -0,0 +1,11 @@
+{
+  "createTime": "2021-06-03T22:09:25.291Z",
+  "lifecycleState": "ACTIVE",
+  "name": "k8s-staging-cluster-api-nested",
+  "parent": {
+    "id": "758905017065",
+    "type": "organization"
+  },
+  "projectId": "k8s-staging-cluster-api-nested",
+  "projectNumber": "766094112860"
+}
```

Comment:
Expected (as well as all of the resources in this dir), this was added in https://github.com/kubernetes/k8s.io/pull/2123